tutorials/Land/Soil/Water/equilibrium_test.jl

```diff
@@ -45,48 +45,48 @@ using ClimateMachine.BalanceLaws:
 
 
 # - define the float type desired (`Float64` or `Float32`)
-FT = Float64;
-
+FT = Float64
```

Comment:
```suggestion
const FT = Float64
```
Maybe it'd be better to just set this constant to avoid the closure issues.